packages/react-composites/src/composites/ChatComposite/file-sharing/FileUploadButton.tsx

```diff
@@ -34,6 +19,11 @@ export interface FileUploadButtonProps {
    * Similar to the `multiple` attribute of the `<input type="file" />` element.
    */
   multiple?: boolean;
+
+  /**
+   * onChange handler for the file upload button.
```

Comment:
Can you say in this comment what is changing in this onChange? (I'm confused how a Button has an `<input>` field)

Reply:
For sure.